mockup/patterns/thememapper/pattern.thememapper.less

```diff
@@ -1,6 +1,6 @@
 @import "@{mockuplessPath}/ui.less";
 /* just be lazy here and include all of bootstrap... */
-@import "@{mockupPath}/filemanager/pattern.filemanager.less";
+@import "@{mockup-patterns-filemanager}";
```

Comment:
It's best practice in mockup to use the aliases when they exist to make it easier to override resources in build tools (and I needed this to override Plone default version with this in my webpack dev env).